taipy/gui/gui.py

```diff
@@ -2907,3 +2907,44 @@ def set_favicon(self, favicon_path: t.Union[str, Path], state: t.Optional[State]
             self._broadcast(
                 "taipy_favicon", url, self._get_client_id() if state else None, message_type=_WsType.FAVICON
             )
+
+    @staticmethod
+    def _add_event_listener(
+        event_name: str,
+        listener: t.Union[
+            t.Callable[[str, t.Dict[str, t.Any]], None], t.Callable[[State, str, t.Dict[str, t.Any]], None]
+        ],
+        with_state: t.Optional[bool] = False,
+    ):
+        try:
+            _Hooks()._add_event_listener(event_name, listener, with_state)
+        except Exception as e:
+            _warn("Hooks: ", e)
```

Comment:
```suggestion
            _warn("Exception in Hook._add_event_listener(): ", e)
```

Reply:
In fact you were right from the start (as usual ?)
All the exceptions are caught and logged in the _Hooks class.
I'll remove the try/catch